index.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Personal-Portfolio</title>
+    <link rel="stylesheet" href="style.css" />
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.15.4/css/all.css"
+    />
+  </head>
+  <body>
+    <header>
+      <div class="my-logo">
+        <nav class="menu-bar">
+          <nav class="logo">Richard Obaseki</nav>
```

Comment:
-[ ] Would you please wrap your logo with an `<a>` tag so that it looks like `<a>Richard Obaseki</a>`  . This is important because it is supposed to redirect you to the homepage.

Reply:
Implemented 👍